Batteries/Data/Array/Lemmas.lean

```diff
@@ -112,9 +112,77 @@ theorem mem_join : ∀ {L : Array (Array α)}, a ∈ L.join ↔ ∃ l, l ∈ L 
   · rintro ⟨s, h₁, h₂⟩
     refine ⟨s.data, ⟨⟨s, h₁, rfl⟩, h₂⟩⟩
 
+/-! ### indexOf? -/
+
```

Comment:
I don't think you're supposed to leave a line between the docstring and the definition

Reply:
Note this isn't a docstring, and it's also following the pattern of the lemmas elsewhere in this file.